dashboard/scripts/archive/import_milestone_logs

```diff
@@ -10,10 +10,9 @@ DB_PASSWORD = ''
 
 DIR = '/mnt/backups/'
 
-c = DBI.connect("DBI:Mysql:#{DB_NAME}:#{DB_HOST}", DB_USER, DB_PASSWORD)
-#c = DBI.connect('DBI:Mysql:dashboard_development:localhost', 'root', '')
-
-#raise c.execute("select now()").inspect
+# c = DBI.connect("DBI:Mysql:#{DB_NAME}:#{DB_HOST}", DB_USER, DB_PASSWORD)
+# c = DBI.connect('DBI:Mysql:dashboard_development:localhost', 'root', '')
+# raise c.execute("select now()").inspect
```

Comment:
Honestly, I have no idea. Mostly, I don't think it is worth spending significant amounts of time trying to find out. There was a lint error (unused variable c), thus the impetus for some change.